examples/CivisML_parallel_training.ipynb

```diff
@@ -173,16 +185,18 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "#### Using CivisML builtin algorithms"
+    "#### Using CivisML built-in algorithms"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "We can start by trying a few different algorithms to fit our classifier. CivisML uses a ModelPipeline object to encapsulate training and predicting. A few important details:\n",
+    "CivisML was built with two goals in mind: make it really fast and easy to try things, and do not force a user to compromise in terms of model quality and flexibility.  To help make iteration fast and easy, we pre-loaded CivisML with a few of our workhorse algorithms, including a regularized logistic regression, a random forest classifier, and an extra trees classifier. As data scientists, we usually want to quickly try a few different models to get our bearings and gain a sense of if a problem will be easy or hard, so a change to only one line of code can swap these different algorithms in and out for each other.\n",
+    "\n",
+    "In the code block below, we start by trying a few different algorithms to fit our classifier. CivisML uses a ModelPipeline object to encapsulate training and predicting. A few important details:\n",
     "\n",
-    "- There are several pre-defined model types you can use to create a pipeline (check the docs above for a full list).\n",
+    "- As mentioned above, CivisML makes a few scikit-learn compatible estimators available. You can see a list of them [here](https://civis-python.readthedocs.io/en/v1.5.2/ml.html#custom-models). \n",
```

Comment:
This link is to an old version of CivisML. How about we link to "latest" instead: https://civis-python.readthedocs.io/en/latest/ml.html#custom-models . I don't like linking to master branch docs, but it will decrease the risk of documentation rot here.